<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora de Números Complexos</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Primeiro número:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Segundo número:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit1NC_PR">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit1NC_PI">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>i</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2NC_PI">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>i</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2NC_PR">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultado:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditResultado">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdicao">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSubtracao">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMultiplicacao">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDivisao">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEhIgual">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>São iguais?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEhDiferente">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>São diferentes?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Calculadora de Números Complexos</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>lineEdit2NC_PI</zorder>
   <zorder>lineEdit1NC_PR</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit1NC_PI</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>lineEdit2NC_PR</zorder>
   <zorder>label_8</zorder>
   <zorder>lineEditResultado</zorder>
   <zorder>pushButtonAdicao</zorder>
   <zorder>pushButtonSubtracao</zorder>
   <zorder>pushButtonMultiplicacao</zorder>
   <zorder>pushButtonDivisao</zorder>
   <zorder>pushButtonEhIgual</zorder>
   <zorder>pushButtonEhDiferente</zorder>
   <zorder>label</zorder>
   <zorder>line</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
